codes/AppEndpointType.ttl

```diff
@@ -32,4 +32,10 @@ idsc:PROCESS_ENDPOINT
     a ids:AppEndpointType;
     rdfs:label "Process endpoint"@en ;
     rdfs:comment "Endpoint is used to start (or control) app processing."@en ;
+.
+
+idsc:USAGE_POLICY_ENDPOINT
+    a ids:AppEndpointType;
+    rdfs:label "usage policy endpoint"@en ;
```

Comment:
Please capitalize the first word `Usage` to follow the other definitions in this file, thanks

Reply:
Fixed in a234eace2004a25f00da9fce1bcf1e406cfae6eb